### PR TITLE
Update 0.5: Get fixes from bee that was added just after my grab due to sods law

### PR DESCRIPTION
**Pull request by Jalleo**
It just works.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -18,9 +18,16 @@
 /code/controllers/subsystem/dbcore.dm @francinum
 /code/controllers/subsystem/timer.dm @francinum
 
-# qwertyquerty
```
Comment (francinum) [inline .github/CODEOWNERS:21]: Drop all modifications to this file.

File: .github/workflows/comp_changelog.yml [hunk 2]
```diff
@@ -34,4 +35,5 @@ jobs:
       - name: "Push"
         uses: ad-m/github-push-action@master
         with:
-          github_token: ${{ secrets.BEEBOT_PAT }}
+          #github_token: ${{ secrets.GITHUB_TOKEN }} Use this instead if you have unprotected branches
```
Comment (francinum) [inline .github/workflows/comp_changelog.yml:38]: @Crossedfall Are our secrets updated to reflect this refactor?

File: .github/workflows/make_changelogs.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   MakeCL:
     runs-on: ubuntu-latest
-    if: github.repository == 'BeeStation/BeeStation-Hornet'
+    if: github.repository == 'BeeStation/BeeStation-Hornet' && !contains(github.event.head_commit.message, '[ci skip]')
```
Comment (francinum) [inline .github/workflows/make_changelogs.yml:10]: Modify to our repository.

File: .github/CODEOWNERS (hunk 1 above)
Comment (Jalleo) [reply .github/CODEOWNERS:21]: Understood.

File: .github/workflows/make_changelogs.yml (hunk 3 above)
Comment (Jalleo) [reply .github/workflows/make_changelogs.yml:10]: There are a few of these that I need to correct actually. I have to go through all the github files.

**Pull request merged**